test/e2e/util/Expect.js

```diff
@@ -1,35 +1,30 @@
 var Expect = module.exports = {};
 
 var _ = require('lodash'),
-    grunt = require('grunt'),
+    fs = require('fs-extra'),
     axe = require('axe-webdriverjs'),
-    protractor = require('protractor');
+    protractor = require('protractor'),
+    textReporter = require('./text-reporter'),
+    junitReporter = require('./junit-reporter');
+
+
+function makeFileName(url) {
+  return './build2/reports/508/junit/' + url.split('://').pop().replace(/[\.\:\/]/g, '-') + _.uniqueId('_') + '.xml';
+}
+
+fs.emptyDirSync('./build2/reports/508');
```

Comment:
Double checking - is this created relative to test/e2e/util, or is it relative to the root okta-signin-widget directory (should output to okta-signin-widget/build2))?